pydantic/decorator.py

```diff
@@ -177,9 +178,7 @@ def build_values(self, args: Tuple[Any, ...], kwargs: Dict[str, Any]) -> Dict[st
 
     def execute(self, m: BaseModel) -> Any:
         d = {k: v for k, v in m._iter() if k in m.__fields_set__}
-        kwargs = d.pop(self.v_kwargs_name, None)
-        if kwargs:
-            d.update(kwargs)
+        var_kwargs = d.pop(self.v_kwargs_name, None) or {}
```

Comment:
```suggestion
        var_kwargs = d.pop(self.v_kwargs_name, {})
```

Reply:
I believe `d[v_kwargs_name]` is actually `None` at this point if no var keywords are provided.

Using `or {}` ensures we have a dictionary, simplifying later code

Reply:
never mind! all tests pass. realise that this is from a model instance, so it will always be a dictionary.

Reply:
When I read `d.pop(self.v_kwargs_name, None) or {}` I thought you wanted to handle the `None` case but I thought it was impossible. For me it was always either unset or a dict hence my suggestion. Sorry if I was mistaken